### ISSUE TITLE
Swamp Waders Ruby and Amethyst effects are "working as intended," meaning they can actually reduce your total Armor/MR/Pierces if your movespeed is fast enough

### ISSUE THREAD

**Issue by DeckardCainRPC** [ChalkyBrush/roshpit-bug-tracker#1445]
with perfect gems, the effects give you 6000 stats, minus 20 for each point of current movespeed. perfectly fine. since minimum movespeed is 100, that means it can give a maximum bonus of 4000 at any given time. neat neat neat.

and of course, if your current movespeed is strictly 300, it will subtract 6000 from this maximum bonus of 6000, leaving 0. no bonus at all. so it is written, He giveth, He taketh away, amen.

HOWEVER

if your movespeed is ever more than 300, having these gems *further penalizes* these stats. at RPC's default and very tolerable 400 movespeed, not only would you forfeit the 6000-point grand prize, you would *lose an additional 2000 points* of whatever the gem affects.

now, this is technically working as intended, because there is no math error involved. the issue is that these gems cannot be used as "emergency stats for when i am slowed very very badly, but which otherwise don't impact me positively or negatively while i'm not being slowed," which is doubly unfortunate considering Ablecore Greaves doesn't really do that anymore either.

furthermore, the boots only reduce movespeed by 250 when equipped, and there are very few other movespeed-reducing effects in RPC that heroes can deliberately apply to themselves, so when used alone, the boots only reduce your movespeed to 150... meaning these dangerous gems, which force you to avoid ever increasing your movespeed beyond the nauseating 100 minimum, also demand *further sacrifices* from you in order to maximize their bonuses. which, let's be fucking honest, in the laughably hypothetical scenario that a player is seriously interested in using these gems, they are NEVER going to fucking walk around our game's gigantic fucking maps at 100 speed - they will ONLY agree to this condition if they can smirk and use a rapid teleportation ability of some sort instead. and even then, that imaginary grinning retard is even dumber than he realizes, because there are much easier and cheaper ways of acquiring 6000 armor/pierces from a single equip slot *without* subjecting your hero the Coach Clearmoon Experience.

so in conclusion here is what i suggest be done about this: just cap the minimum "bonus" from these gems at 0. that's all. lol. thanks for watching, like and subscribe
